src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -110,12 +110,12 @@ public UnorderedCallAssertion MustHaveHappened(Repeated repeatConstraint)
             return new UnorderedCallAssertion(this.manager, this.Matcher, description.Builder.ToString(), repeatConstraint);
         }
 
-        private void EnsureRuleAdded()
+        private void EnsureRuleIsAdded()

```

Comment:
I hesitated beween `EnsureRuleWasAdded` and `EnsureRuleIsAdded`... I found the former a bit strange, so I picked the latter. Let me know if you think differently (I don't really care either way)